workflow/rules/common.smk

```diff
@@ -409,15 +419,29 @@ def get_consensus_input(wildcards):
     elif is_activated("remove_duplicates"):
         return "results/dedup/{sample}.bam"
     else:
-        aligner = "star" if get_sample_datatype(wildcards.sample) == "rna" else "bwa"
+        if get_sample_datatype(wildcards.sample) == "rna":
+            aligner = "star"
+        elif get_sample_datatype(wildcards.sample) == "dna" & is_activated(
+            "ref/pangenome"
+        ):
+            aligner = "vg"
+        else:
+            aligner = "bwa"
```

Comment:
Same code as above, create new function `get_aligner(wildcards)` and use in both blocks.

Reply:
and below as well